intezer_sdk/analysis.py

```diff
@@ -407,7 +419,12 @@ def from_latest_analysis(cls,
                                                                  url=url,
                                                                  aggregated_view=True,
                                                                  api=api)
-        analyses_ids = [report['analysis_id'] for report in analysis_history_url_result.all()]
+
+        analyses_ids = [
+            report['analysis_id'] for report in analysis_history_url_result.all()
+            if _domain_contains(report['submitted_url'], url)
```

Comment:
Let's talk, not sure i understand why we need the fix here and not in the actual search